dist/esm/index.mjs

```diff
@@ -1,6 +1,7 @@
 import * as React from 'react';
 import PlyrJS from 'plyr';
 import useAptor from 'react-aptor';
+import PropTypes from 'prop-types';
```

Comment:
If you want to add prop-types, please add here https://github.com/chintan9/plyr-react/blob/master/src/index.tsx